src/main/java/com/theironyard/invitator/Person.java

```diff
@@ -0,0 +1,140 @@
+package com.theironyard.invitator;
+
+import java.sql.*;
+import java.util.ArrayList;
+
+/**
+ * Created by melmo on 12/22/16.
+ */
+public class Person {
+    private Integer personId;
+    private String name;
+    private String phone;
+    private String email;
+    private String photoUrl;
+    private Boolean invited;
+
+    public Person(Integer personId, String name, String phone, String email, String photoUrl, Boolean invited) {
+        this.personId = personId;
+        this.name = name;
+        this.phone = phone;
+        this.email = email;
+        this.photoUrl = photoUrl;
+        this.invited = invited;
+    }
+
+    public Person(String name, String phone, String email, String photoUrl, Boolean invited) {
+        this.personId = 0;
+        this.name = name;
+        this.phone = phone;
+        this.email = email;
+        this.photoUrl = photoUrl;
+        this.invited = invited;
+    }
+
+    /* *
+     * Saves a single 'Person' object to 'people' table in DB 'invitation'
+     * */
+    public void savePerson() throws SQLException {
```

Comment:
This is redundant.  `Person.savePersion()` would be less redundant with `Person.save()` given that the save is in the person class.